fluent-kubernetes-loomsystems.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluentd-loomsystems
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-loomsystems
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentd-loomsystems
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: fluentd
+        image: loomsystems/fluentd-kubernetes-loomsystems:v0.12.43-loomsystems
+        imagePullPolicy: Always
+        env:
+          - name:  FLUENT_UID
+            value: "0"
+          - name:  LOOM_RECIEVER_HOST
+            value: "129.213.38.169"
+          - name:  LOOM_RECIEVER_SSL_PORT
+            value: "9999"
```

Comment:
Are these ports correct? They are non-standard ports.

Reply:
fixed, these setting will be defined by the user